src/renderers/WebGLRenderer.js

```diff
@@ -315,6 +315,12 @@ function WebGLRenderer( parameters ) {
 
 	this.vr = vr;
 
+	// Multiview
+
+	var multiview = new WebGLMultiview( _this, _gl, _gl.getContextAttributes() );
+
+	this.multiview = multiview;
```

Comment:
I don't think we should expose this.

Reply:
Yep, fixed. It was used to access the number of views on the WebGLProgram, but I just changed it to get the number of views from the rendertarget directly https://github.com/mrdoob/three.js/pull/16316/files?file-filters%5B%5D=.js#diff-58e8247d21441c70f4f66e068eac5bfaR334